lib/manager/github-actions/extract.ts

```diff
@@ -31,6 +31,7 @@ export function extractPackageFile(content: string): PackageFile | null {
       continue;
     }
 
+    line = line.replaceAll('"', '');
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll, `replaceAll` was only introduced in Node 15, while Renovate supports Node 14 still: https://github.com/renovatebot/renovate/blob/f0a1b64b15963114fd3976f8a77e4ff4134748c6/package.json#L123

Also, are you sure that `"` wouldn't be used within any strings validly?

Reply:
well i didnt know that, thanx , 
so its better to add the check into the regex as far as i can see.
ill add  `\"?`  to the regex on both sides

Reply:
Trim it and remove any leading and trailing quotation marks?

Reply:
Assuming you can isolate the value